.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+**What type of PR is this? (check applicable) 🚧**
+
+- [ ] ♻️ Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 👷 Optimization
+- [ ] 📝 Documentation Update
+- [ ] 🔖 Release
+- [ ] 🚩 Other
+
+**Describe your PR request 🔥**
+
+<!-- A clear and consise description of your PR request-->
+
+**Mobile & Desktop Screenshots/Recordings 📷**
+
+<!-- 
+UI changes should be accompanied by screenshots or recordings. Please upload/link them here. 
+Guide to linking images here:
+https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#uploading-assets
+-->
+
+**Related Issues 🩹**
+
+<!-- 
+Example: 
+Resolves #123
+Closes #456
+
+Guide to linking issues here:
+https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
+-->
+
+**Added to documentation? 📝**
+
+- [ ] 📜 readme
+- [ ] 📜 other
+- [ ] 🙅 no documentation needed
+
```

Comment:
**suggestion:** remove

Reply:
Should we not link the pull request with the issue

Reply:
The `Development` section under `Milestone` section on the side bar to the right of each pull requests. This section can link the issue.

Reply:
Very cool!